src/item.cpp

```diff
@@ -2298,6 +2245,69 @@ std::string item::display_name(unsigned int quantity) const
     return string_format("%s%s%s", name.c_str(), side.c_str(), qty.c_str());
 }
 
+std::string item::get_item_tags(bool parens) const
+{
+    std::string rtn;
+    std::stringstream ret;
+    ret.str("");
```

Comment:
Converted `ret` to type `std::ostringstream`, removed `ret.str("");` line, and removed unnecessary use of the `std::string rtn` variable, instead simply returning `ret.str()`.